### DISCUSSION TITLE
VR and FMGS

### DISCUSSION THREAD

**Discussion by Simpilot2021** [flybywiresim/aircraft#3009] [Help] [answered]
Hi
Starting VR deletes the F-PLN in the FMGS. Any time you switch on VR via STRG+TAB the Flightplan disappears and need to reload.
Any ideas or someone else with experience?
Thanks
Andreas

**Comment by Inaxair** [comment, marked as answer]
Hello @Simpilot2021 

We are aware of the issue, and we are working on a fix for this.

Best regard,
Inaxair from FlyByWire Simulations

**Comment by Simpilot2021** [comment]
Sounds great,
Thank you
Andreas

**Reply by ppanther69** [reply]
Hi,

 The proposed fix is to turn off VFR map within the in-flight options menu using the slider option, (see custom tool bar / reset panels).

https://forums.flightsimulator.com/t/losing-flight-plan-when-switching-between-2d-and-vr/344059

I tried this, and it works.

**Comment by Simpilot2021** [comment]
Thanks for this proposal.
I tried it and it works.
BR
Andreas
